### PR TITLE
adds method and test for fetching a project by ID

### PR DESCRIPTION
**Pull request by amyrebecca**

### CODE REVIEW

File: lib/panoptes/client/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module Panoptes
   class Client
-    VERSION = "0.3.6".freeze
+    VERSION = "0.3.7".freeze
```
Comment (marten) [inline lib/panoptes/client/version.rb]: I get it, but generally speaking bumping a version in a PR is frowned upon.

**Comment by marten** [comment]
Can you make sure to update the changelog as well?

**Comment by amyrebecca** [comment]
Yep, I didn't know about the changelog but I'll do that.

File: lib/panoptes/client/version.rb (hunk 1 above)
Comment (amyrebecca) [reply lib/panoptes/client/version.rb]: I'm confused--how else do we version the gem?

**Comment by marten** [comment]
Usually version changes happen in a commit to master, not bundled with a feature PR. Largely because two PRs would collide (or in a typical OS project, contributors don't know about the release schedule)

- Marten

**Comment by amyrebecca** [comment]
Ah, that makes a lot of sense. Thank you! :)

**Pull request merged**